### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by levisweetbreu** [mindearth/mobilkit#3]
@takayabe0505 I am reviewing mobilkit for submission to JOSS, and I have a few notes about installation.

1. Adding instructions or a link for Dask installation would be helpful to users.
2. It would be useful to include installation instructions for all operating systems. I use Windows and had to do a few extra and different steps than what is listed in the documentation.
3. When testing the mobilkit install in python, I got an error message `AttributeError: module 'pandas.core.strings' has no attribute 'StringMethods'` and I had to run the command `pip install --upgrade pandas "dask[complete]"` to solve the issue and address the importation error. I'm not sure if this was an issue with my Dask installation or the mobilkit installation but it would be useful to look into.

https://github.com/openjournals/joss-reviews/issues/5201

**Comment by ubi15** [comment]
Hi @levisweetbreu:
1. ok, will add the link to Dask installation in the README and documentation;
2. We do not have a Windows machine to test on, could you briefly summarize the extra steps you had to go through to have it running?
3. Probably it was a Pandas/Dask version mismatch. We rely on a specific Dask version (`dask[full]==2022.4.1`) as we encountered some bugs in the `groupby` functions in the following versions. I will update the `requirements.txt` file to mirror this issue.

Will implement these changes as soon as I have time and mention this issue in the commit.

Thanks for your inputs and let me know if you have other comments.

**Comment by ubi15** [comment]
Hi @levisweetbreu I did the following in [71a548c](https://github.com/mindearth/mobilkit/commit/71a548cc7e773cdc8a8273d1030800333a0d31e0) and [8d65cf9](https://github.com/mindearth/mobilkit/commit/8d65cf95afee9ef71782b9a1c28702642db2158a):
1. Actually the separate Dask installation was not needed, changed the text accordingly;
2. Added the Windows instructions, let me know if I got your suggestions right;
3. Updated the `requirements.txt ` even though they are not used by the packaging system and updated `Dask` and `Pandas` to avoid that issue.

I will later release the `0.2.7` versions including these changes if you do not see other issues.
